forum/templatetags/filter_img.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from BeautifulSoup import BeautifulSoup, Tag
+from os.path import basename, splitext
+
+register = template.Library()
+
+
+@register.filter(is_safe=True)
+def replace_img(string):
+    soup = BeautifulSoup(string)
```

Comment:
bs is quite slow IIRC... :|

why don't we process the stuff on input and remove any HTTP(-not-s) links there?